tests/functional/relation_tests/test_relation_type_change.py

```diff
@@ -34,31 +40,86 @@ def error_message(self) -> str:
 
 relations = [
     Model(models.VIEW, "view"),
-    Model(models.TABLE, "table"),
-    Model(models.DYNAMIC_TABLE, "dynamic_table"),
+    Model(models.TABLE, "table", "default"),
+    # to be activated upon merge of dynamic table support PR
```

Comment:
these will be uncommented when we reconcile merge conflicts with Mike's dynamic table PR